background/index.ts

```diff
@@ -112,6 +110,12 @@ async function main() {
         return
       }
 
+      const block = await node.getBlock(userOpReceipt.receipt.blockNumber)
+
+      if (!block) {
+        return
+      }
+
```

Comment:
I feel block timestamp is not that important to record in receipt. If you really want to show it on page, you can just query by provider in page.

Reply:
No, In our extension context, `blockTimestamp` isn't essential compared to `creationAt`;
if Bundler RPC can fetch it, it can stay, but it currently requires Node RPC.

Next commit I will remove `blockTimestamp`.